packages/syft/src/syft/__init__.py

```diff
@@ -96,7 +96,7 @@
 
 sys.path.append(str(Path(__file__)))
 
-logger.add(sink=sys.stderr, level="CRITICAL")
+logger.start()
```

Comment:
Q: Why did change to start() ?

Reply:
https://github.com/OpenMined/PySyft/blob/dev/packages/syft/src/syft/logger.py
Our custom  logger class does not have start method I think

Reply:
Ignore this, The changes you made to logger file were down :sweat_smile:

Reply:
did not see it before

Reply:
i wanted to be able to pause logging temporarily - so i created start() and stop() methods